### ISSUE TITLE
The two test signalhubs on now sh are not available anymore

### ISSUE THREAD

**Issue by epcau** [mafintosh/signalhub#42]
I am trying to solve by hosting one myself on heroku, I can see it running and listening to 8080 but I don't understand which address I should use to subscribe, the heroku default one doesn't work

**Comment by LucCADORET** [comment]
I think that heroku instances have a $PORT variable, so maybe just `signalhub listen -p $PORT` would make it publicly available ?

**Comment by epcau** [comment]
Yes, after that post I've been doing more research and actually managed to get the signalhub/server running on heroky. Too bad it's not the best on free heroku since after some minutes I get the process to sleep automatically to save resources

**Comment by LucCADORET** [comment]
I think that amazon has cheap EC2 "micro" instances, and I think those are not going to sleep (though you'd have to verify)

**Comment by soyuka** [comment]
I think that @RangerMauve has some running instances, but then again if you have high load just use your own :)
